File: app/src/main/java/com/ddpc/ggway/ui/fragment/search/ViewSearchFragment.java
package com.ddpc.ggway.ui.fragment.search;

import com.ddpc.ggway.ui.MainView;

import java.util.List;

/**
 * Created by diha- on 10.02.2018.
 */

public interface ViewSearchFragment extends MainView {
    void refreshCategoryState(List<Object> list);
}

File: app/src/main/java/com/ddpc/ggway/ui/fragment/search/ViewSearchFragment.kt
package com.ddpc.ggway.ui.fragment.search

import com.ddpc.ggway.ui.MainView

/**
 * Created by diha- on 10.02.2018.
 */

interface ViewSearchFragment : MainView {
    fun refreshCategoryState(list: List<Any>)
}
